src/nvim/CMakeLists.txt

```diff
@@ -83,6 +87,9 @@ file(GLOB NVIM_HEADERS *.h)
 file(GLOB XDIFF_SOURCES xdiff/*.c)
 file(GLOB XDIFF_HEADERS xdiff/*.h)
 
+list(APPEND TS_SOURCES ${TREESITTER_PATH}/lib/src/lib.c)
```

Comment:
I don't see `lib.c` in the PR anymore.

Reply:
@justinmk It was never part of the PR, `${TREESITTER_PATH}` is expected to be a checked-out (but not otherwise built) tree-sitter repo dir. Previously it just assumed tree-sitter was checked out as a sibling dir, now the `.deps` build will check out a fixed commit as `./.deps/build/src/`